PWG/EMCAL/EMCALtasks/AliEmcalCorrectionTask.cxx

```diff
@@ -1321,6 +1325,7 @@ Bool_t AliEmcalCorrectionTask::Run()
     component->SetCentralityBin(fCentBin);
     component->SetCentrality(fCent);
     component->SetVertex(fVertex);
+    component->SetUseMergedBCs(fDoUseMergedBCs);
```

Comment:
This attribute doesn't belong in the correction task. Each one of the other values that are set here vary event-by-event, but this one doesn't. It should be set individually for each component using YAML.